scene/3d/audio_stream_player_3d.cpp

```diff
@@ -98,33 +98,41 @@ static const Vector3 speaker_directions[7] = {
 void AudioStreamPlayer3D::_calc_output_vol(const Vector3 &source_dir, real_t tightness, AudioStreamPlayer3D::Output &output) {
 	unsigned int speaker_count; // only main speakers (no LFE)
 	switch (AudioServer::get_singleton()->get_speaker_mode()) {
-		default: //fallthrough
```

Comment:
This was... interesting :D

Reply:
@zwostein Did you add this on purpose? IINM, it was forcing `speaker_count` to be `2` regardless of the speaker mode, since the switch would always go through `default`, then fallthrough to `case AudioServer::SPEAKER_MODE_STEREO: speaker_count = 2;` and break.

Reply:
Yes, the fallthrough was intentional - probably unnecessary, though. The default case would only have matched if no other case matches. Might be a bit unconventional ;)

Reply:
Oh, looking at the change: If get_speaker_mode() returns a value not handled in the switch, speaker_count will be uninitialised.

Reply:
Right, while reading if I though of the GDScript `match` statement where the default case is a catch-all, and cases are processed sequentially. But here indeed default is only used in absence of match.

`get_speaker_mode()` returns an enum value and all 4 possible values of the enum are handled, so I don't think we need to worry about potential uninitialization. If a 5th enum value was added, compilers would throw a warning (treated as error on CI) of it's not handled in the switch.